torch/nn/modules/__init__.py

```diff
@@ -44,7 +44,7 @@
     'Tanhshrink', 'RReLU', 'L1Loss', 'NLLLoss', 'KLDivLoss', 'MSELoss', 'BCELoss', 'BCEWithLogitsLoss',
     'NLLLoss2d', 'PoissonNLLLoss', 'CosineEmbeddingLoss', 'CTCLoss', 'HingeEmbeddingLoss', 'MarginRankingLoss',
     'MultiLabelMarginLoss', 'MultiLabelSoftMarginLoss', 'MultiMarginLoss', 'SmoothL1Loss', 'GaussianNLLLoss',
-    'HuberLoss', 'SoftMarginLoss', 'CrossEntropyLoss', 'Container', 'Sequential', 'ModuleList', 'ModuleDict',
+    'HuberLoss', 'SoftMarginLoss', 'CrossEntropyLoss','Container', 'Sequential', 'ModuleList', 'ModuleDict',
```

Comment:
This change also seems like an accident

Reply:
But I don't see that mistake in my file.